### PR TITLE
Feature/activity button

### PR DESCRIPTION
**Pull request by michellekaplan7**
## Type of change made:
 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [X] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Styling- no new features

## Detailed Description

We added a nav, which includes 3 buttons (activity, sleep, and hydration). When you click on the activity button, a pop up card appears and the background blurs out (added a class in the css). The pop up card includes fields to update activity. You can click the back button and it will take you back to the page, or you can click submit which will post the data to the endpoint. We added the eventHandler, showActivityForm and buildActivityPostObject functions to the scripts.js and added the postActivityData method to the api-controller class. 

We ran into some issues formatting the date, but figured it out by pulling in moment and in our creation of our activity object we split at the hyphens and joined with a slash! 

Files modified:
 - [X] index.html
 - [X] styles.scss
 - [ ] base.scss
 - [X] scripts.js
 - [ ] User.js
 - [ ] User-repo.js
 - [ ] Activity.js
 - [ ] Hydration.js
 - [ ] Sleep.js
 - [X] api-controller.js
 
 Other files:

### CODE REVIEW

File: src/api-controller.js [hunk 1]
```diff
@@ -15,16 +15,29 @@ class ApiController {
...
 
- getSleepData() {
-   let url = `${this.rootUrl}/sleep/sleepData`
-   return fetch(url).then(response => response.json());
- }
+  getSleepData() {
+    let url = `${this.rootUrl}/sleep/sleepData`
+    return fetch(url).then(response => response.json());
+  }
+
```
Comment (carflor) [inline src/api-controller.js:22]: Nice setup for the other button fetches!

File: src/css/style.scss [hunk 2]
```diff
@@ -248,3 +271,11 @@ h1 {
   font-weight: bold;
   font-family: sans-serif;
 }
+
+.hidden {
+  visibility: none;
+}
```
Comment (carflor) [inline src/css/style.scss:277]: Cannot remember what we are using this hidden visibility for? Did we end up using only the blur?

Comment (ttarlov) [reply src/css/style.scss:277]: @carflor I dont think we are using the hidden class for anything at this point.

File: package.json [hunk 3]
```diff
@@ -32,6 +32,8 @@
     "webpack-dev-server": "^3.1.14"
   },
   "dependencies": {
-    "jquery": "^3.4.1"
+    "flatpickr": "^4.6.3",
```
Comment (ttarlov) [inline package.json:35]: i think we can remove flatpickr from the dependencies. Ill see how we can do that.

**Comment by ttarlov** [comment]
@carflor @michellekaplan7 We can merge this in as is or if Michelle wants to delete that hidden class and push the branch back up we can merge then. Either way works.

**Pull request merged**